File: Arrays/Missing_in_array.py
class Solution:
    
    # Note that the size of the array is n-1
    def missingNumber(self, n, arr):
        
        sum = (n*(n+1))//2
        arr_sum = 0
        for i in range(n-1):
            arr_sum += arr[i]
        return sum-arr_sum

File: Arrays/Missing_in_Array.cpp
class Solution {
  public:

    // Note that the size of the array is n-1
    int missingNumber(int n, vector<int>& arr) {

        // Your code goes here
        // first pass : negate the array
        int sum = (n*(n+1))/2;
        int arr_sum =0;
        for(int i=0;i<n-1;i++){
            arr_sum+=arr[i];
        }
        return sum-arr_sum ;
    }
};
